src/pages/userPage/personalData/AvatarAndProgressStatic.tsx

```diff
@@ -206,16 +267,15 @@ const AvatarAndProgressStatic: React.FC<AvatarAndProgressStaticProps> = (
       </div>
       {UserDistinctions.map((dist) => (
         <div className="distinctions">
-          {(dist.distinction.name?.length > nameMaxLength) ?
+          {dist.distinction.name?.length > nameMaxLength ? (
             <Tooltip title={dist?.distinction.name}>
               <h2>
-                {dist.distinction.name.slice(0,54) + "..."} №{dist.number}
+                {dist.distinction.name.slice(0, 54) + "..."} №{dist.number}
```

Comment:
here there is one type of tabulation for h2, belove for h2 another

Reply:
fixed